core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -41,17 +48,33 @@ angular.module('oppia').controller('RteHelperModalController', [
     $timeout(function() {
       $scope.modalIsLoading = false;
     });
-
+    $scope.isRteMathExpressionEditor = false;
```

Comment:
Do not start a variable name with `is` (or verb in general)

Reply:
done